test/ngAria/ariaSpec.js

```diff
@@ -750,6 +750,18 @@ describe('$aria', function() {
     });
   });
 
+  describe('actions when bindRoleForClick is set to false', function() {
+      beforeEach(configAriaProvider({
+          bindRoleForClick: false
+      }));
+      beforeEach(injectScopeAndCompiler);
+
+      it('should not add a button role', function() {
+        compileElement('<radio-group ng-click="something"></radio-group>');
+        expect(element.attr('role')).toBeUndefined();
+      });
+  });
```

Comment:
It matches the rest of the file. I didn't want to introduce whitespace changes, especially to make the whole file consistent. But if it's important, I can....